README.md

```diff
@@ -32,3 +32,16 @@ If you want to check the secrets you've already set you can run
 ```bash
 dotnet user-secrets list
 ```
+
+### CRUD and Auth API
+
+There is a high chance you will need info from a backend API route that requires authorization - for that you will need to use Auth API.
+
+1. Import UserContext and Auth API from UserProvider: `import { UserContext, authApi } from '../contexts/UserProvider';`
+1. Get the global user state: `const [user, setUser] = useContext(UserContext);`
+1. Call any CRUD method you need:
+   1. GET method: `authApi(user).get(<url>)`
+   1. POST method (default behavior is upsert\*) `authApi(user).post(<url>, <body>)` It is important that `<body>` matches the database model
+   1. DELETE method `authApi(user).post(<url>, <body>, "DELETE")` Just as in the previous method, it is important that `<body>` matches the database model
+
```

Comment:
Good documentation, though we probably should have a separate documentation file for this instead of README

Reply:
Good point, we will have to look into separating code documentation and project documentation